### ISSUE TITLE
First cell appears behind the textInputBar instead of above it

### ISSUE THREAD

**Issue by briankracoff** [slackhq/SlackTextViewController#334]
The first UITableViewCell in the UITableView is behind the `textInputBar` when the UIViewController appears instead of having a top content inset greater than 0. It will only fix itself when I tap on the `textView` (bringing up the keyboard) and then tap off of it (dismissing the keyboard).

This is after upgrading from `v1.7.2` to `v1.8`. I bisected around and looks like https://github.com/slackhq/SlackTextViewController/commit/3ce0f7bf57c62989e00ae156fff925736cb9c16d breaks it.

**Comment by dzenbot** [comment]
Thanks. Will look into it.

**Comment by briankracoff** [comment]
Found out what's going on. I was adding a `UIRefreshControl` to the `UITableView` to the top as usual (which will put it at the bottom since `inverted == true`). 

However, I called `refreshControl.beginRefreshing()` in `viewDidLoad` so it'll show as refreshing when the user comes into the screen. I'm guessing `UIRefreshControl` does some `contentInset` changes that messes with the `contentInset` changes that you guys do.

Moving `refreshControl.beginRefreshing()` to `viewDidAppear` fixes the issue.

Feel free to close if this is too specific of a bug.
